recipes/_user.rb

```diff
@@ -95,14 +96,12 @@
   else
     directory build_user_home do
       owner node['omnibus']['build_user']
-      group node['omnibus']['build_user_group']
       action :create
     end
   end
 else
   directory build_user_home do
     owner node['omnibus']['build_user']
-    group node['omnibus']['build_user_group']
```

Comment:
See https://github.com/chef-cookbooks/omnibus/compare/master...sensu:fix/group-properties-windows for our implementation. @schisamo @atomic-penguin